tensorflow_addons/layers/adaptive_pooling.py

```diff
@@ -183,24 +233,132 @@ def __init__(
         self.data_format = conv_utils.normalize_data_format(data_format)
         self.reduce_function = reduce_function
         self.output_size = conv_utils.normalize_tuple(output_size, 2, "output_size")
+        self.output_size_x, self.output_size_y = self.output_size
```

Comment:
We can either access through `self.output_size[0]` and `self.output_size[1]` or unpack it in `call` so do not set them as attribute.